versions.gradle

```diff
@@ -10,178 +10,17 @@ androidVersions.buildTools = '30.0.2'
 androidVersions.targetSdk = 30
 ext.androidVersions = androidVersions
 
-ext.deps = [:]
```

Comment:
I think we should remove this file, we should locate the androidVersions in the gradle.properties file or in the main build.gradle file. What do you think?